### PR TITLE
made paasta start/stop deploy-group aware

### PR DESCRIPTION
**Pull request by mjksmith**
This fixes a bug where paasta start and stop wouldn't work on tasks using deploy groups.

### CODE REVIEW

File: paasta_tools/cli/cmds/start_stop_restart.py [hunk 1]
```diff
@@ -139,19 +131,48 @@ def paasta_start_or_stop(args, desired_state):
...
+            service=service,
+            instance=instance,
+            cluster=cluster,
+            soa_dir=soa_dir,
+            load_deployments=False,
+        )
+    else:
+        print "A service config other than type 'marathon' or 'chronos' was loaded!"
+        print "'%s' is not currently supported by paasta %s" % (instance_type, desired_state)
+        sys.exit(3)
```
Comment (solarkennedy) [inline paasta_tools/cli/cmds/start_stop_restart.py]: Yuck. We already have almost literally this same code in 
paasta_tools/cli/cmds/local_run.py:550
Can you move that to utils and share that instead?

File: paasta_tools/generate_deployments_for_service.py [hunk 2]
```diff
@@ -139,6 +139,7 @@ def get_deploy_group_mappings(soa_dir, service, old_mappings):
     remote_refs = remote_git.list_remote_refs(git_url)
 
     for control_branch, deploy_group in deploy_group_branch_mappings.items():
+        print control_branch, deploy_group
```
Comment (solarkennedy) [inline paasta_tools/generate_deployments_for_service.py]: probably don't need this right?

**Comment by solarkennedy** [comment]
We have the itest for this now, and it passes

https://travis-ci.org/Yelp/paasta/jobs/105775872#L208-L224

So that gives credit that this does work as intended.

I would like @EvanKrall to look at this as well. My main concern is the whole `paasta-` thing again, but if the itests pass and generate deployments can read it back out, then I say go for it.

Can you add a deploy_group aware assertion in the deployments.json itest?

**Comment by mjksmith** [comment]
AFAIK the deployments.json itest already has an instance that uses deploy groups.

**Comment by solarkennedy** [comment]
afaict this is good to go. The only think I'm worried about is the whole `paasta-` thing still.

Lets ship this carefully together. Merge when you are ready.

**Pull request merged**